metadata-ingestion/source_docs/clickhouse.md

```diff
@@ -83,6 +83,7 @@ Note that a `.` is used to denote nested fields in the YAML recipe.
 
 | Field                       | Required | Default                                                                    | Description                                                                                                                                                                             |
 |-----------------------------|----------|----------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `connect_uri`                  |          |          | URI of database to connect to. See https://docs.sqlalchemy.org/en/14/core/engines.html#database-urls. Takes precedence over other connection parameters.
```

Comment:
Can we move this parameter to below the native connector configurations? ie below username + password + port since this is exposing internal implementation detail.

I'd also prefer if we name this "sqlalchemy_uri" to make it explicit the type of URI we are exposing here.. Thoughts?

Reply:
Agree - both of those changes seem reasonable

Reply:
@jjoyce0510  I can update the order in the docs, but renaming the parameter would make it inconsistent with the current generic sqlalchemy source https://datahubproject.io/docs/metadata-ingestion/source_docs/sqlalchemy